apiserver/SecuringImplementation.md

```diff
@@ -0,0 +1,69 @@
+# Securing API server
+
+Currently the KubeRay API server is directly exposed to the internet with no authentication/authorization. To protect its endpoint we need to introduce security.
```

Comment:
I think this statement is only true if the user does deploy the apiserver on publicly accessable cluster. A private VPC deployment or on premise cluster deployment is not automatically exposed to the internet.

Reply:
Sure